docs/OpenShift GitOps CLI User Guide.md

```diff
@@ -0,0 +1,654 @@
+# OpenShift GitOps CLI User Guide
+OpenShift GitOps `argocd` CLI enables users to manage OpenShift GitOps from a terminal. Using the CLI tool, users can manage ArgoCD resources (Application, ApplicationSets, AppProjects, User accounts, GPG keys etc) from a client terminal.
+
+**Note:** Both the archives and the RPMs contain the `argocd` executable binary file.
+
+**WARNING** 	
+Red Hat OpenShift GitOps with the `argocd` CLI tool is a Technology Preview feature only.
+
+**Tip:** This guide assumes you have a grounding in the tools that OpenShift GitOps is based on. Please read [about Red Hat OpenShift GitOps](https://docs.openshift.com/gitops/1.11/understanding_openshift_gitops/about-redhat-openshift-gitops.html) to learn about these tools.
+
+## Installing OpenShift GitOps CLI (argocd)
+Use the CLI tool to manage Red Hat OpenShift GitOps from a terminal. You can install the CLI tool on different platforms.
+
+### Installing the Red Hat OpenShift GitOps CLI on Linux using an RPM
+For Red Hat Enterprise Linux (RHEL) version 8 or version 9, you can install the Red Hat OpenShift GitOps CLI as an RPM.
+#### Prerequisites
+You have an active OpenShift Container Platform subscription on your Red Hat account.
+You have root or sudo privileges on your local system.
+#### Procedure
+- Register with Red Hat Subscription Manager:
```

Comment:
@anandf Steps should be in an ordered/numbered list as they are a sequence of steps.